src/lumigo_tracer/libs/awslambdaric/bootstrap.py

```diff
@@ -2,44 +2,30 @@
 Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 """
 
-import warnings
+import importlib
+import sys
 
-from lumigo_tracer.libs.lambda_runtime_exception import FaultException
-
-with warnings.catch_warnings():
-    warnings.filterwarnings("ignore", category=DeprecationWarning)
-    import imp
-
-ERROR_LOG_LINE_TERMINATE = "\r"
-ERROR_LOG_IDENT = "\u00a0"  # NO-BREAK SPACE U+00A0
+from .lambda_runtime_exception import FaultException
 
 
 def _get_handler(handler):
     try:
         (modname, fname) = handler.rsplit(".", 1)
     except ValueError as e:
         fault = FaultException(
-            FaultException.MALFORMED_HANDLER_NAME, "Bad handler '{}': {}".format(handler, str(e))
+            FaultException.MALFORMED_HANDLER_NAME,
+            "Bad handler '{}': {}".format(handler, str(e)),
         )
         return make_fault_handler(fault)
 
-    file_handle, pathname, desc = None, None, None
```

Comment:
just making sure: this is all copied from the latest version, correct?

Reply:
Yes, awslambdaric 2.0.8